ql/src/test/org/apache/hadoop/hive/ql/txn/compactor/CompactorTest.java

```diff
@@ -181,7 +184,7 @@ protected Table newTable(String dbName, String tableName, boolean partitioned,
     table.setTableName(tableName);
     table.setDbName(dbName);
     table.setOwner("me");
-    table.setSd(newStorageDescriptor(getLocation(tableName, null), sortCols));
+    table.setSd(newStorageDescriptor(getLocation(tableName, null).toString(), sortCols));
```

Comment:
minor: getLocation returning String was more natural here

Reply:
Restored the String return type https://github.com/apache/hive/pull/3514/commits/bff0fe3d4b941b01b89bc4d853d93841116c8bfd